Classes/YTPlayerView.m

```diff
@@ -509,24 +602,63 @@ + (NSString *)stringForPlayerState:(YTPlayerState)state {
       return kYTPlayerStatePausedCode;
     case kYTPlayerStateBuffering:
       return kYTPlayerStateBufferingCode;
-    case kYTPlayerStateQueued:
+    case kYTPlayerStateCued:
       return kYTPlayerStateCuedCode;
     default:
       return kYTPlayerStateUnknownCode;
   }
 }
 
+#pragma mark - WKNavigationDelegate
+
+- (void)webView:(WKWebView *)webView
+decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
+decisionHandler:(void (^)(WKNavigationActionPolicy))decisionHandler {
+  NSURLRequest *request = navigationAction.request;
+  if ([request.URL.host isEqual: self.originURL.host]) {
```

Comment:
When would this condition ever be true? Since we're setting "self.originURL" to the bundleID, that will always result in a non-standard URL and I couldn't think of a scenario for which this "if" statement would evaluate to "true".